src/llmq/quorums_utils.h

```diff
@@ -89,6 +89,14 @@ class CLLMQUtils
         }
         return HexStr(vBytes);
     }
+    template <typename CacheType>
+    static void InitQuorumsCache(CacheType& cache)
+    {
+        for (auto& llmq : Params().GetConsensus().llmqs) {
```

Comment:
Can llmq be const?

Reply:
You could add it, but `llmq` is already const because `GetConsensus()` returns a const reference so `llmqs` is const and it would be kind of redundant.